### ISSUE TITLE
launching with iOS9

### ISSUE THREAD

**Issue by zwacky** [phonegap/ios-deploy#171]
having issues with deploying to an iOS9 device.

```
------ Debug phase ------
Starting debug of J82AP 'iPad' (8a497d9129f608f961fd237fadf0e995ba43a28b) connected through USB...
[ !! ] Unable to locate DeveloperDiskImage.dmg.
[ !! ] This probably means you don't have Xcode installed, you will need to launch the app manually and logging output will not be shown!

>> Error code 253 for command: ios-deploy with args: -d,-b,/Users/zwacky/data/mc-webapp/mobile/platforms/ios/build/device/JustWatch.app
>> 
```

Info:

**1. sw_vers -productVersion**
10.10.5

**2. ios-deploy -V**
1.5.0

**3. xcodebuild -version**
Xcode 6.4
Build version 6E35b

**4. xcode-select --print-path**
/Applications/Xcode.app/Contents/Developer

**5. gcc --version**
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 6.1.0 (clang-602.0.53) (based on LLVM 3.6.0svn)
Target: x86_64-apple-darwin14.5.0
Thread model: posix

**6. lldb --version**
lldb-330.0.48

**Comment by shazron** [comment]
First try restarting your phone (on issue #155, someone reported that it worked after).
If that fails, try this: https://github.com/phonegap/ios-deploy/issues/155#issuecomment-132029683

**Comment by shaunmccabe** [comment]
I am seeing this with iOS 9 (I am running the version of ios-deploy from issue #155 cited above, with --verbose):

The call is: ios-deploy -i "<my_real_device_id>" -b /tmp/iosbuilds/MyApp.app -n -r --verbose
...
[ 52%] CreatingStagingDirectory
[ 57%] ExtractingPackage
[ 60%] InspectingPackage
[ 60%] TakingInstallLock
[ 65%] PreflightingApplication
[ 65%] InstallingEmbeddedProfile
[ 65%] InstallingEmbeddedProfile
[ 70%] VerifyingApplication
2015-09-17 19:10:40.593 ios-deploy[25603:942474] [ !! ] Error 0xe80000da: unknown. AMDeviceSecureInstallApplication(0, device, url, options, install_callback, 0)

**Comment by shazron** [comment]
This looks like a code signing issue. Try to build in Xcode for device, right click on the Product in the Project Navigator and Show in Finder. Deploy that

**Comment by shaunmccabe** [comment]
Yes yes, that worked. I just need to add the appropriate codesign call to the end of my command line build script. Thanks!

**Comment by yulia-falcore** [comment]
"Unable to locate DeviceSupport directory" happens for me too. Restarting the iPhone didn't help. The debugging version which mentioned in [#155 (comment)](https://github.com/phonegap/ios-deploy/issues/155#issuecomment-132029683) is no longer available.

Any suggestions?
![screen shot 2017-06-28 at 12 54 09](https://user-images.githubusercontent.com/25635051/27631374-edb1593a-5c00-11e7-96d4-45734b584029.png)

**Comment by mstorsjo** [comment]
FWIW, I also started getting this. In a setup that used to work about a month ago, I'm suddenly getting this error (with an iPhone 5S running 10.3.2, where deployments from Xcode works fine). I've tried deploying apps from Library/Developer/Xcode/DerivedData as well, so it shouldn't be an issue with how the app is packaged.

**Comment by mstorsjo** [comment]
I was able to fix the issue for myself, with the fix in pull request #299.
